conf/modules/logger_sd_spi_direct.xml

```diff
@@ -0,0 +1,61 @@
+<!DOCTYPE module SYSTEM "module.dtd">
+
+<module name="logger_sd_spi_direct" dir="loggers">
+  <doc>
+    <description>Direct SPI SD Logger that saves pprzlog messages to SD Card.</description>
+    <configure name="SDLOGGER_DIRECT_SPI" value="SPI1|SPI2|SPI3|SPI4|SPI5|SPI6" description="Port to which the SD Card is connected."/>
+    <configure name="SDLOGGER_DIRECT_SPI_SLAVE" value="SPI_SLAVE1|SPI_SLAVE2|SPI_SLAVE3|SPI_SLAVE4|SPI_SLAVE5|SPI_SLAVE6" description="Port to which the SD Card is connected."/>
+    <configure name="LOGGER_CONTROL_SWITCH" value="RADIO_AUX2"/>
```

Comment:
Naming should be consistent here.... e.g. name them all starting with `SDLOGGER_DIRECT_` or so